src/components/shared/Entity/Create/Config.tsx

```diff
@@ -0,0 +1,114 @@
+import { Box, CircularProgress, Collapse, Typography } from '@mui/material';
+import ConnectorTiles from 'components/ConnectorTiles';
+import ExistingEntityCards from 'components/shared/Entity/ExistingEntityCards';
+import {
+    useExistingEntity_connectorName,
+    useExistingEntity_createNewTask,
+    useExistingEntity_hydrated,
+} from 'components/shared/Entity/ExistingEntityCards/Store/hooks';
+import useEntityCreateNavigate from 'components/shared/Entity/hooks/useEntityCreateNavigate';
+import useGlobalSearchParams, {
+    GlobalSearchParams,
+} from 'hooks/searchParams/useGlobalSearchParams';
+import useBrowserTitle from 'hooks/useBrowserTitle';
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { EntityWithCreateWorkflow } from 'types';
+
+interface Props {
+    title: string;
+    entityType: EntityWithCreateWorkflow;
+}
+
+function EntityCreateConfig({ title, entityType }: Props) {
+    useBrowserTitle(title);
+
+    const [connectorId] = useGlobalSearchParams([
+        GlobalSearchParams.CONNECTOR_ID,
+    ]);
+
+    const navigateToCreate = useEntityCreateNavigate();
+
+    // Existing Entity Store
+    const existingEntityStoreHydrated = useExistingEntity_hydrated();
+    const createNewTask = useExistingEntity_createNewTask();
+    const connectorName = useExistingEntity_connectorName();
+
+    const [showConnectorTiles, setShowConnectorTiles] = useState<
+        boolean | null
+    >(null);
+
+    useEffect(() => {
+        if (typeof connectorId === 'string') {
+            setShowConnectorTiles(false);
+        } else {
+            setShowConnectorTiles(true);
+        }
+    }, [connectorId]);
```

Comment:
I am confused on why this was done this way as it seems way over engineered. Can you put in a comment on why this was done.

Reply:
You authored this block of code. It originally lived in the shared `EntityCreate` component and was copied and pasted over when the component was spliced. Agreed, that it can be simplified.